pos_menu/static/src/js/pos.js

```diff
@@ -0,0 +1,26 @@
+odoo.define('pos_menu', function(require) {
+    var models = require('point_of_sale.models');
+
+    models.load_models([{
+        model: 'pos.tag',
+        field: [],
+        domain: null,
+        loaded: function(self, tags) {
+            self.tags = [];
+            // save tags of current POS
+            tags.forEach(function(tag){
+                if ($.inArray(self.config.id, tag.pos_ids) !== -1) {
+                    self.tags.push(tag);
+                }
+            });
+            // add new domain for product
+            self.models.forEach(function(model) {
+                if (model.model === "product.product") {
+                    model.domain.push(['tag_ids', 'in', self.config.tag_ids]);
```

Comment:
:-1: 
When tag list is empty, it must load all products

Reply:
@yelizariev not problem)